static/src/javascripts/projects/commercial/modules/messenger/viewport.js

```diff
@@ -0,0 +1,76 @@
+// @flow
+import detect from 'lib/detect';
+import fastdom from 'lib/fastdom-promise';
+import { register } from 'commercial/modules/messenger';
+
+let w = window;
+let iframes = {};
+let iframeCounter = 0;
+let taskQueued = false;
+
+const lastViewportRead = () => fastdom.read(() => detect.getViewport());
```

Comment:
Probably my lack of understanding, but does this value need to be updated, is it OK as a `const`?